tests/topotests/static_vrf/r1/frr.conf

```diff
@@ -0,0 +1,20 @@
+debug static route
+
+interface r1-eth0 vrf red
+  ip address 1.1.1.1/24
```

Comment:
Please no public IPs (in this case Cloudflare). See: https://github.com/FRRouting/frr/blob/master/doc/developer/topotests.rst (^f + `Always use IPv4`).

Reply:
To be honest, using these ranges makes tests less comprehensible, it's just harder to understand the addressing when there's only 3 /24 subnets. But I changed the addresses according to the rules.